muffnn/autoencoder/autoencoder.py

```diff
@@ -176,8 +177,7 @@ def _set_up_graph(self):
         second_layers \
             = list(self.hidden_units[::-1][1:]) + [self.input_layer_size_]
         for i, layer_sz in enumerate(second_layers):
-            if self.dropout is not None:
```

Comment:
yeah, true.  

At first, i thought that doing this wasn't necessary because of the check that TF has [here](https://github.com/tensorflow/tensorflow/blob/r0.11/tensorflow/python/ops/nn_ops.py#L1184), but after another look I think it's still needed since we have `keep_prob` as a placeholder.